pkg/istio-agent/agent.go

```diff
@@ -520,38 +493,66 @@ func fileExists(path string) bool {
 
 // Find the root CA to use when connecting to the CA (Istiod or external).
 func (a *Agent) FindRootCAForCA() (string, error) {
-	var rootCAPath string
-
-	if a.cfg.CARootCerts == security.SystemRootCerts {
-		return "", nil
-	} else if a.cfg.CARootCerts != "" {
```

Comment:
Is this option no longer being used?

Reply:
I am not sure if this one is still in use. If we can confirm that this option is no longer being used, we can remove this part.